src/js/components/reviewData/ReviewDataComments.jsx

```diff
@@ -0,0 +1,66 @@
+/**
+ * ReviewDataComments.jsx
+ * Created by Alisa Burdeyny 11/21/16
+ **/
+
+import React from 'react';
+import ReviewDataCommentsDropdown from './ReviewDataCommentsDropdown.jsx'
+import ReviewDataCommentsTextfield from './ReviewDataCommentsTextfield.jsx'
+
+import * as ReviewHelper from '../../helpers/reviewHelper.js';
+
+export default class ReviewDataComments extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            currentFile: "A",
+            fileComments: props.comments,
+            saveState: ""
+        };
+    }
+
+    changeFile(newFile) {
+        var fileComment = document.getElementById("submission-review-comments").value.trim();
```

Comment:
Apologies if this is a bad suggestion, but it seems like we shouldn't be using `document` in these functions. Is this where `refs` should be used?